### DISCUSSION TITLE
How can i ignore languages key when search in translated data ?

### DISCUSSION THREAD

**Discussion by mahmoudahmed2222** [spatie/laravel-translatable#403] [Q&A]
i have languages table stored in DB when i search for key (en) to return  english  languages i get all languages without filtration .
how can i ignore languages key when search for text ?

screen shoot for languages table 
https://prnt.sc/SWkTzULgP5P5

**Comment by mahmoudahmed2222** [comment]
@freekmurze  can you help me please ?

**Reply by donnyxray** [reply]
How about this?

```
// Search only English names
$lang = Languages::where('name->en', '=', 'French')->first();

// Return only English name
$name = $lang->getTranslation('name','en');
```

**Reply by mahmoudahmed2222** [reply]
i want to search in all languages not only on specific language

**Comment by ArtMin96** [comment]
@mahmoudIsProgramer Have you found a solution?
